coalahtml/_coalahtml/app/controllers/files.js

```diff
@@ -13,12 +13,11 @@ angular.module('coalaHtmlApp')
       $scope.fileBack = fileName.split("/").slice(0, -1).join('/');
       if ($rootScope.FILES.hasOwnProperty(fileName)) {
         $scope.fileType = "dir";
-        $scope.fileContents = {};
-        for(var i_file = 0;i_file < $rootScope.FILES[fileName].length;i_file += 1) {
-          var dirFile = $rootScope.FILES[fileName][i_file];
-          var dirFileType = $rootScope.FILES.hasOwnProperty(dirFile) ? "dir": "file";
-          $scope.fileContents[dirFile] = dirFileType;
-        }
+        $scope.fileContents = [];
+        $rootScope.FILES[fileName].forEach(function(file){
```

Comment:
`forEach` should be added in dde0dd67ae29e7a04dafbf5b6410f5f1a5447489. Here it's unrelated.